src/crimson/os/seastore/root_meta.h

```diff
@@ -0,0 +1,79 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+
+#pragma once
+
+#include "crimson/os/seastore/cached_extent.h"
+
+namespace crimson::os::seastore {
+
+struct RootMetaBlock : LogicalCachedExtent {
+  using meta_t = std::map<std::string, std::string>;
+  using Ref = TCachedExtentRef<RootMetaBlock>;
+  static constexpr size_t SIZE = 4096;
+  static constexpr int MAX_META_LENGTH = 1024;
+
+  explicit RootMetaBlock(ceph::bufferptr &&ptr)
+    : LogicalCachedExtent(std::move(ptr)) {
+    set_meta(meta_t{});
```

Comment:
Not sure if this is correct -- looks this means the meta will be set to empty unconditionally even if it 
isn't empty, for example, in the read path.

`set_meta(meta_t{})` should be called only when the extent is freshly allocated.

Reply:
I think this cstor is called only when the extent is freshly allocated. In the read path, extents are also freshly allocated through this cstor first and then filled with data on disks. Am I missing something?

Reply:
Maybe we should add a "fresh_alloc_marker" to this cstor in CachedExtent, what do you think? @cyx1231st

Reply:
How about moving `set_meta(meta_t{})` to `TM::init_root_meta()` right after `alloc_non_data_extent<RootMetaBlock>()`?

This way, we don't need to rely on specific read implementions.

Reply:
Yeah, will do:-)

Reply:
All done, please take a look, again. Thanks:-)